pubsublite/spark-connector/spark_streaming_to_pubsublite_example.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+
+
+def spark_streaming_to_pubsublite(
+    project_number: int, location: str, topic_id: str
+) -> None:
+    # [START pubsublite_spark_streaming_to_pubsublite]
+    from pyspark.sql import SparkSession
+    from pyspark.sql.types import BinaryType, StringType
+
+    # TODO(developer):
+    # project_number = 11223344556677
+    # location = "us-central1-a"
+    # topic_id = "your-topic-id"
+
+    spark = SparkSession.builder.appName("write-app").master("yarn").getOrCreate()
+
+    # Create a RateStreamSource that generates consecutive numbers with timestamps:
+    # |-- timestamp: timestamp (nullable = true)
+    # |-- value: long (nullable = true)
+    sdf = spark.readStream.format("rate").option("rowsPerSecond", 1).load()
+
+    sdf = (
+        sdf.withColumn("key", (sdf.value % 5).cast(StringType()).cast(BinaryType()))
```

Comment:
1. I saw you are only using 1 partition, then no matter how different ordering key is, they all lands on that single partition
2. can you just cast to binary directly?

Reply:
1. Changed to 2.
2. Casting directly to binary is not allowed.

Reply:
for 2, can you add some description or comments? Do you have any references for this. External users are going to read this and write their script so it would be helpful to explain this more.